wordpress-mobile/WordPressCom-Stats-iOS
# Preventing evil offline crash

### Testing:
- Launch the StatsDemo Project
- Scroll all the way down to the *Followers* section
- Turn off networking (LTE / Wifi)
- Tap over the `View All` row

Please, verify that, as a result, the demo project doesn't crash anymore.

Needs Review: @astralbodies 
Thanks!!!
